apps/i18n/applab/my_mm.json

```diff
@@ -1,14 +1,82 @@
 {
   "catActions": "လုပ်​ဆောင်ချက်များ",
   "catControl": "Loops",
+  "catEvents": "ဖြစ်စဥ်များ",
   "catLogic": "Logic",
   "catMath": "သင်္ချာ",
   "catText": "စာ",
   "catVariables": "Variables",
   "designWorkspaceHeader": "Workspace: ",
   "dropletBlock_drawImage_param1": "×",
+  "dropletBlock_radioButton_param2": "အုပ်စု",
+  "dropletBlock_setBlue_param3": "အပြာရောင်တန်ဖိုး",
+  "dropletBlock_setChecked_param1": "စစ်ဆေးပြီးပြီ။",
+  "dropletBlock_setGreen_param3": "အစိမ်းရောင်တန်ဖိုး",
+  "dropletBlock_setKeyValue_param1_description": "သိမ်းဆည်းထားရမယ့် အချက်အလက်တွေပေါ့။",
+  "dropletBlock_setKeyValueSync_param1": "တန်ဖိုး",
+  "dropletBlock_setKeyValueSync_param1_description": "သိမ်းဆည်းထားရမယ့် အချက်အလက်တွေပေါ့။",
+  "dropletBlock_setNumber_description": "ဖော်ပြထားတဲ့အရာအတွက် နံပါတ်တန်ဖိုး သတ်မှတ်ပေးပါ။",
   "dropletBlock_setRGB_param3": "အနီ",
   "dropletBlock_setRGB_param4": "အစိမ်း",
   "dropletBlock_setRGB_param5": "အပြာရောင်",
-  "dropletBlock_substring_param1": "ပြီးပါပြီ"
+  "dropletBlock_substring_param1": "ပြီးပါပြီ",
+  "dropletBlock_turnRight_description": "လိပ်ကို တိကျသေချာသောထောင့်အနေအထား၏ ညာဘက်သို့လှည့်ပါ။ လိပ်၏အနေအထားမှာ တူညီနေရပါမယ်။",
+  "dropletBlock_turnRight_param0_description": "ညာဘက်ထောင့်သို့ ရွှေ့ရန်",
+  "dropletBlock_turnTo_description": "လိပ်၏လမ်းကြောင်းကို တိကျသေချာသောထောင့်အနေအထားသို့ပြောင်းပါ။ ၀ သည် အပေါ်ဘက်၊ ၉၀ သည် ညာဘက်၊ ၁၈၀ သည် အောက်ဘက်၊ ၂၇၀ သည် ညာဘက်ဖြစ်ပါသည်။",
+  "dropletBlock_turnTo_param0_description": "လိပ်၏ဦးတည်ချက်သတ်မှတ်ရန် ထောင့်အသစ်။",
+  "dropletBlock_updateRecord_description": "အပ်လပ်(ဘ်)၏ အချက်အလက်ကို အသုံးပြု၍ ဇယားကွက်တွင်ပေးထားသောမှတ်တမ်းကို နာမည်အသစ်ဖြင့် အဆင့်မြင့်ပါ။ မှတ်တမ်းသည် ၎င်း၏ အိုင်ဒီအကွက်ဖြင့် ထူးခြာစွာခွဲထွက်နေရပါမည်။  ခေါ်ဆိုမှုပြီးဆုံးပါက ပြန်လည်ခေါ်ဆိုခြင်းလုပ်ဆောင်ချက်ကို ခေါ်ပြီးပြီဖြစ်ပါသည်။",
+  "dropletBlock_updateRecord_param2_description": "အဆင့်မြင့်တင်သောမှတ်တမ်း()ပြီးဆုံးသောအခါတွင် မတူညီစွာပြန်ခေါ်သည့်လုပ်ဆောင်ချက်တစ်ခု။",
+  "dropletBlock_write_description": "သတ်မှတ်ထားသော စာသားကို စာရွက်စာတမ်း၏ အောက်ခြေတွင် ထပ်မံ၍ပေါင်းထည့်ပါ။ စာသားကိုလည်း HTML ပုံစံအဖြစ်ပြုလုပ်နိုင်သည်။",
+  "dropletBlock_write_param0_description": "သင်ထည့်သွင်းလိုသော စာသား သို့မဟုတ် HTMLသင့်ဆော့ဖ်ဝဲလ်၏အောက်ခြေတွင် ထပ်မံ၍ထည့်သွင်းသည်။",
+  "foundDevices": "အမှတ်စဉ်တွင် အသုံးပြုနိုင်သော စက်ကို ရှာမတွေ့ပါ။\nတွေ့ရှိသော စက်များ- {စက် စာရင်း}",
```

Comment:
Indeed.

Reply:
I think there are redaction/restoration issues also within this translation.

Reply:
Yeah a lot of the Burmese translations had issues. I went through the translators history and fixed them all. I will do another down+out to get the updates. Thanks for pointing that out.